ontology/generate_ontology.py

```diff
@@ -110,7 +112,7 @@ class Inspector(object):
 
     """ Class that includes methods for querying an RDFS/OWL ontology """
 
-    def __init__(self, uri, language=''):
+    def __init__(self, uri, comments=False, language=''):
```

Comment:
is there any call to this init using `language` only and no comments? If it does, it will fail since it will assign the value to the new `comments` param. If this call exists we will have to move comments to the end:
`def __init__(self, uri, language='', comments=False)`.

in case there's no such initialization of this class, we are good and no need to change anything

Reply:
Actually language parameter is not used. I will remove it. All the ontologies won't need comments except CLO since a lot of CLO terms don't have definitions, we check theirs comments besides definition. Thanks!